### PR TITLE
Rohan fiton

### PR DESCRIPTION
**Pull request by rohnnie**
Added google auth feature

### CODE REVIEW

File: FitOn/settings.py [hunk 1]
```diff
@@ -1,19 +1,68 @@
 import os
 from pathlib import Path
 from datetime import timedelta
+import boto3
+import os
```
Comment (KarthikVV24-11-1999) [inline FitOn/settings.py:5]: os is imported twice, it's already imported above

File: FitOn/settings.py [hunk 2]
```diff
@@ -1,19 +1,68 @@
...
 # See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = 'django-insecure-iqw@@a4osoerv=_))5ipw&kthcyr@v55xwz#=sse!13()+s#l_'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
+
+ALLOWED_HOSTS = ['*']
```
Comment (KarthikVV24-11-1999) [inline FitOn/settings.py:32]: ALLOWED_HOSTS is already present in line number 67 below

File: FitOn/templates/profile.html [hunk 3]
```diff
@@ -24,11 +24,122 @@
                     </div>
                 </div>             
 
+<<<<<<< HEAD
```
Comment (KarthikVV24-11-1999) [inline FitOn/templates/profile.html:27]: Some merge conflicts issues are not properly resolved before pushing

File: FitOn/urls.py [hunk 4]
```diff
@@ -14,52 +14,37 @@
...
-    path('login/', login, name='login'),
-    path('deactivate/', deactivate_account, name='deactivate_account'),
-    path('deactivate/confirm/', confirm_deactivation, name='confirm_deactivation'),
-    path('logout/', login, name='logout'),
-    path('reset-password/', password_reset_request, name='password_reset_request'),
-    path('reset-password/<str:user_id>/<str:token>/', password_reset_confirm, name='password_reset_confirm'),
-    path('reset-password/done/', password_reset_done, name='password_reset_done'),
-    path('reset-password/complete/', password_reset_complete, name='password_reset_complete'),
-    path('profile/', profile_view, name='profile'),
-    path('upload_profile_picture/', upload_profile_picture_view, name='upload_profile_picture'),
```
Comment (KarthikVV24-11-1999) [inline FitOn/urls.py:56]: It should be "upload_profile_picture_view" like in the main branch

**Review by KarthikVV24-11-1999 — CHANGES_REQUESTED** [review]
> Multiple issues are observed due to improperly resolving the merge conflicts locally

File: FitOn/settings.py (hunk 1 above)
Comment (rohnnie) [reply FitOn/settings.py:5]: Resolved

File: FitOn/settings.py (hunk 2 above)
Comment (rohnnie) [reply FitOn/settings.py:32]: Resolved

File: FitOn/templates/profile.html (hunk 3 above)
Comment (rohnnie) [reply FitOn/templates/profile.html]: resolved

File: FitOn/urls.py (hunk 4 above)
Comment (rohnnie) [reply FitOn/urls.py:56]: resolved

**Review by KarthikVV24-11-1999 — APPROVED** [review]

**Pull request merged**